### PR TITLE
Server-2047 | Simplified flyway & postgres chart migration

### PR DESCRIPTION
**Pull request by atulsingh0**
# Description
* Removed Postgres chart migrations Instructions as those will be executed as migration script
* `circle-migrator` job will be automatically deleted if completed successfully. 

# Reasons
[SERVER-2047](https://circleci.atlassian.net/browse/SERVER-2047)

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: [https://circleci.com/docs/style/style-guide-overview](https://circleci.com/docs/style/style-guide-overview).

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [ ] Break up walls of text by adding paragraph breaks.
- [ ] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [ ] Keep the title between 20 and 70 characters.
- [ ] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [ ] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [ ] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

File: jekyll/_cci2/server/installation/migrate-from-server-3-to-server-4.adoc [hunk 1]
```diff
@@ -114,18 +104,51 @@ It should be - <domain-name>:4647
...
+Migrating schema "public" to version "0002 - drop domain migrations"
+DEBUG: Executing SQL: DROP TABLE IF EXISTS public.domain_migrations
+DEBUG: 0 rows affected
+DEBUG: Successfully completed migration of schema "public" to version "0002 - drop domain migrations"
+DEBUG: Schema History table "public"."schema_version" successfully updated to reflect changes
+Successfully applied 1 migration to schema "public", now at version v0002 (execution time 00:00.101s)
+DEBUG: Memory usage: 12 of 15M
+----
+
+Else, contact with your circleci point of contact for further instructions.
```
Comment (jbialy) [inline jekyll/_cci2/server/installation/migrate-from-server-3-to-server-4.adoc:133]:
```suggestion
Else, reach out to your circleci point of contact for further instructions.
```

File: jekyll/_cci2/server/installation/migrate-from-server-3-to-server-4.adoc [hunk 2]
```diff
@@ -114,18 +104,51 @@ It should be - <domain-name>:4647
...
+[#check-migrator-job-status]
+=== 3. Check migrator job status
+Check if migrator job status if exists -
+[source,shell]
+----
+kubectl -n <namespace> get job/circle-migrator
+----
+
+If job does not exists then check the pod log `circle-migrator.log` in `log` directory and pod log can be is look like below -
+[source,shell]
```
Comment (circlery) [inline jekyll/_cci2/server/installation/migrate-from-server-3-to-server-4.adoc:117]: is shell highlighting the best here?

File: jekyll/_cci2/server/installation/migrate-from-server-3-to-server-4.adoc [hunk 3]
```diff
@@ -114,18 +104,51 @@ It should be - <domain-name>:4647
...
+
+[#check-migrator-job-status]
+=== 3. Check migrator job status
+Check if migrator job status if exists -
+[source,shell]
+----
+kubectl -n <namespace> get job/circle-migrator
+----
+
+If job does not exists then check the pod log `circle-migrator.log` in `log` directory and pod log can be is look like below -
```
Comment (circlery) [inline jekyll/_cci2/server/installation/migrate-from-server-3-to-server-4.adoc:116]:
```suggestion
If the job does not exist, check the pod's logs `circle-migrator.log` in the `log` directory. An example of the logs can be found below.
```

File: jekyll/_cci2/server/installation/migrate-from-server-3-to-server-4.adoc [hunk 4]
```diff
@@ -114,18 +104,51 @@ It should be - <domain-name>:4647
...
+Migrating schema "public" to version "0002 - drop domain migrations"
+DEBUG: Executing SQL: DROP TABLE IF EXISTS public.domain_migrations
+DEBUG: 0 rows affected
+DEBUG: Successfully completed migration of schema "public" to version "0002 - drop domain migrations"
+DEBUG: Schema History table "public"."schema_version" successfully updated to reflect changes
+Successfully applied 1 migration to schema "public", now at version v0002 (execution time 00:00.101s)
+DEBUG: Memory usage: 12 of 15M
+----
+
+Else, reach out to your circleci point of contact for further instructions.
```
Comment (circlery) [inline jekyll/_cci2/server/installation/migrate-from-server-3-to-server-4.adoc:133]:
```suggestion
For any other concerns, please reach out to your CircleCI point of contact.
```

Do we actually even need this sentence?

**Review by circlery — CHANGES_REQUESTED** [review]

**Review by circlery — APPROVED** [review]
Thanks for the changes! LGTM.

File: jekyll/_cci2/server/installation/migrate-from-server-3-to-server-4.adoc [hunk 5]
```diff
@@ -114,18 +104,48 @@ It should be - <domain-name>:4647
...
+
+[#check-migrator-job-status]
+=== 3. Check migrator job status
+Check if migrator job status if exists -
+[source,shell]
+----
+kubectl -n <namespace> get job/circle-migrator
+----
+
+If the job does not exist, check the pod's logs `circle-migrator.log` in the `log` directory. An example of the logs can be found below.
```
Comment (rosieyohannan) [inline jekyll/_cci2/server/installation/migrate-from-server-3-to-server-4.adoc:116]:
```suggestion
If the job does not exist, check the pod's logs (`circle-migrator.log`) in the `log` directory. An example of the logs can be found below:
```

**Review by rosieyohannan — APPROVED** [review]
Thank you @atulsingh0!

**Pull request merged**